src/print.rs

```diff
@@ -142,7 +142,13 @@ pub fn module(module_name: &str, args: ArgMatches) {
 }
 
 pub fn get_module(module_name: &str, context: Context) -> Option<String> {
-    modules::handle(module_name, &context).map(|m| m.to_string())
+    Some(
```

Comment:
<img src="https://user-images.githubusercontent.com/24731903/147854851-f9322f70-4e48-45ac-ae93-2ea29ffa0a28.png" alt="image" width="200" style="max-width: 100%">

`get_module` used to call `modules::handle` but the handling for `custom`/`env_var` is only in `handle_prompt`, which caused tests using `ModuleRenderer` to fail. So I made a hack to have it go through `handle_prompt` but there might be a cleaner way.